scripts/download_chromium.js

```diff
@@ -1,48 +1,46 @@
 const path = require('path');
-const child_process = require('child_process');
-const Puppeteer = require('puppeteer');
-const { PUPPETEER_REVISIONS } = require('../node_modules/puppeteer/lib/cjs/puppeteer/revisions')
+const {
+    BrowserPlatform,
+    Browser,
+    install,
+    resolveBuildId,
+} = require('@puppeteer/browsers');
 
 const archArg = process.argv[2];
 let [
-  // Should be one of linux, mac, win32, win64 as per options in BrowserFetcher but we reuse the same arch string
-  // as for grpc download (ie darwin-x64-unknown) so we need to transform it a bit
-  platform,
-  arch
+    // Should be one of linux, mac, win32, win64 as per options in BrowserFetcher but we reuse the same arch string
+    // as for grpc download (ie darwin-x64-unknown) so we need to transform it a bit
+    platform,
+    arch,
 ] = archArg.split('-');
 
 if (platform === 'win32' && arch === 'x64') {
-  platform = 'win64'
+    platform = BrowserPlatform.WIN64;
 }
 
 if (platform === 'darwin') {
-  platform = 'mac'
+    platform = BrowserPlatform.MAC;
 }
```

Comment:
I think this can be removed, in the `@puppeteer/browsers` doc, it's said that it'auto-detected: https://pptr.dev/browsers-api/browsers.installoptions#:~:text=platform,Auto%2Ddetected.

Reply:
But what if we explicitly need to build binaries for specific arch? E.g. I'm on arm64 and want to build and package the plugin for Windows?